frontend/cypress/integration/featureFiles/wizard_istio_config.feature

```diff
@@ -216,8 +220,9 @@ Feature: Kiali Istio Config page
     Then the preview button should be disabled
 
 @wizard-istio-config
-  Scenario: Create multiple K8s Gateways with colliding hostnames and port combinations and check for a reference
-    When user clicks in the "K8sGateway" Istio config actions
+  Scenario: Create multiple K8s Gateways with colliding hostnames and port combinations and check for a reference. Then delete one of them and the reference should be gone.
```

Comment:
this scenario is big and might be a bit fragile. I have no problem with merging it right now. If will be stable, we can keep it, if not we will refactor that later.